salt/tests/unit/modules/files/test_metalk8s_monitoring.yaml

```diff
@@ -0,0 +1,169 @@
+alertmanager_api_helper:
+  - _id: nominal
+    endpoint: &alertmanager_endpoint
+      ip: '10.0.0.1'
+      ports:
+        web: 1234
+    route: api/v1/alerts
+    resp_body:
+      status: success
+      data: some result
+    result: some result
+    called_with:
+      args: [GET, http://10.0.0.1:1234/api/v1/alerts]
```

Comment:
```suggestion
      args: [GET, 'http://10.0.0.1:1234/api/v1/alerts']
```
Otherwise I end up with a traceback:
```
======================================================= ERRORS ========================================================
________________________ ERROR collecting salt/tests/unit/modules/test_metalk8s_monitoring.py _________________________
salt/tests/unit/modules/test_metalk8s_monitoring.py:21: in <module>
    YAML_TESTS_CASES = yaml.safe_load(fd)
.tox/unit-tests/lib/python2.7/site-packages/yaml/__init__.py:162: in safe_load
    return load(stream, SafeLoader)
.tox/unit-tests/lib/python2.7/site-packages/yaml/__init__.py:114: in load
    return loader.get_single_data()
.tox/unit-tests/lib/python2.7/site-packages/yaml/constructor.py:74: in get_single_data
    node = self.get_single_node()
ext/_yaml.pyx:707: in _yaml.CParser.get_single_node
    ???
ext/_yaml.pyx:725: in _yaml.CParser._compose_document
    ???
ext/_yaml.pyx:776: in _yaml.CParser._compose_node
    ???
ext/_yaml.pyx:890: in _yaml.CParser._compose_mapping_node
    ???
ext/_yaml.pyx:774: in _yaml.CParser._compose_node
    ???
ext/_yaml.pyx:851: in _yaml.CParser._compose_sequence_node
    ???
ext/_yaml.pyx:776: in _yaml.CParser._compose_node
    ???
ext/_yaml.pyx:890: in _yaml.CParser._compose_mapping_node
    ???
ext/_yaml.pyx:776: in _yaml.CParser._compose_node
    ???
ext/_yaml.pyx:890: in _yaml.CParser._compose_mapping_node
    ???
ext/_yaml.pyx:774: in _yaml.CParser._compose_node
    ???
ext/_yaml.pyx:853: in _yaml.CParser._compose_sequence_node
    ???
ext/_yaml.pyx:905: in _yaml.CParser._parse_next_event
    ???
E   ScannerError: while scanning a plain scalar
E     in "/home/alexandre/work/scm/metalk8s/master/salt/tests/unit/modules/files/test_metalk8s_monitoring.yaml", line 13, column 19
E   found unexpected ':'
E     in "/home/alexandre/work/scm/metalk8s/master/salt/tests/unit/modules/files/test_metalk8s_monitoring.yaml", line 13, column 23
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=============================================== 1 error in 1.75 seconds ===============================================
ERROR: InvocationError for command /home/alexandre/work/scm/metalk8s/master/.tox/unit-tests/bin/pytest salt/tests/unit (exited with code 2)
_______________________________________________________ summary _______________________________________________________
ERROR:   unit-tests: commands failed
```

Reply:
Interesting, worked for me :sweat_smile:  I'll change this